manifests/params.pp

```diff
@@ -132,6 +132,7 @@
   $https_proxy         = '',
   $priority            = '10',
   $port                = '80',
+  $selinux_context     = 'system_u:object_r:httpd_sys_content_t',
```

Comment:
Lint is giving a `manifests/selinux.pp:25:trailing_whitespace:ERROR:trailing whitespace found`

Can you run puppet-lint -f to fix this or delete the whitespace? 👍 @mlosapio